.changeset/short-jobs-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@docs/storybook': patch
```

Comment:
Since this is just a docs update and doesn't actually need to trigger any releases we can just leave this as an empty changeset, which looks like this with no content:

```
---
---
```

This will allow it to pass our branch protection policies (that require a changeset's for everything) without needing to trigger a release step.